test/run_envoy_bazel_coverage.sh

```diff
@@ -41,32 +41,40 @@ echo "    VALIDATE_COVERAGE=${VALIDATE_COVERAGE}"
 # projects that want to run coverage on a different/combined target.
 # Command-line arguments take precedence over ${COVERAGE_TARGET}.
 if [[ $# -gt 0 ]]; then
-  COVERAGE_TARGETS=("$@")
+  COVERAGE_ARGS=("$@")
 elif [[ -n "${COVERAGE_TARGET}" ]]; then
-  COVERAGE_TARGETS=("${COVERAGE_TARGET}")
+  COVERAGE_ARGS=("${COVERAGE_TARGET}")
 else
-  COVERAGE_TARGETS=(//test/...)
+  COVERAGE_ARGS=(//test/...)
 fi
 
 if [[ "${FUZZ_COVERAGE}" == "true" ]]; then
   # Filter targets to just fuzz tests.
-  _targets=$(bazel query "attr('tags', 'fuzz_target', ${COVERAGE_TARGETS[*]})")
+  _targets=$(bazel query "attr('tags', 'fuzz_target', ${COVERAGE_ARGS[*]})")
   COVERAGE_TARGETS=()
   while read -r line; do COVERAGE_TARGETS+=("$line"); done \
       <<< "$_targets"
   BAZEL_BUILD_OPTIONS+=(
       "--config=fuzz-coverage"
       "--test_tag_filters=-nocoverage")
 else
+  COVERAGE_TARGETS=$COVERAGE_ARGS
```

Comment:
From CI:

```
2021-04-28T17:34:59.4251043Z Shellcheck: test/run_envoy_bazel_coverage.sh
2021-04-28T17:34:59.5961884Z 
2021-04-28T17:34:59.5963221Z In test/run_envoy_bazel_coverage.sh line 61:
2021-04-28T17:34:59.5963741Z   COVERAGE_TARGETS=$COVERAGE_ARGS
2021-04-28T17:34:59.5964956Z   ^--------------^ SC2178: Variable was used as an array but is now assigned a string.
2021-04-28T17:34:59.5966965Z                    ^------------^ SC2128: Expanding an array without an index only gives the first element.
2021-04-28T17:34:59.5967411Z 
2021-04-28T17:34:59.5967791Z For more information:
2021-04-28T17:34:59.5968516Z   https://www.shellcheck.net/wiki/SC2128 -- Expanding an array without an ind...
2021-04-28T17:34:59.5970085Z   https://www.shellcheck.net/wiki/SC2178 -- Variable was used as an array but...
```

Reply:
Yep, but after fixing there's one more problem that running bazel coverage twice doesn't aggregate the coverage profile.

Reply:
Should be possible to merge with genhtml after copying them out